app/assets/javascripts/helpers/date.js

```diff
@@ -0,0 +1,60 @@
+// Convert JS Date object into UNIX timestamp
```

Comment:
I would suggest to use existing date library like Moment.js or Date.js.
Implementing its own helpers is usually time consuming and error-prone approach.